src/main/frontend/components/property/value.cljs

```diff
@@ -654,34 +658,35 @@
                  excluded-options (remove (fn [e] (contains? exclude-ids (:block/uuid e))) options)]
              excluded-options)
 
+           (and (= property-type :class) (nil? classes))
+           (get-all-classes-f)
+
            (seq classes)
            (->>
             (mapcat
              (fn [class]
                (if (= :logseq.class/Root (:db/ident class))
-                 (model/get-all-classes repo {:except-root-class? true
-                                              :except-private-tags? (not (contains? #{:logseq.property/template-applied-to} (:db/ident property)))})
+                 (get-all-classes-f)
```

Comment:
I think this whole branch can be removed since we don't care about special Root behavior anymore

Reply:
Agreed! I'll remove it soon.